deploy/Chart/templates/rp/deployment.yaml

```diff
@@ -69,6 +69,10 @@ spec:
         volumeMounts:
         - name: config-volume
           mountPath: /etc/config
+        {{- if eq .Values.global.awsIRSA.enabled true }}
+        - name: aws-iam-token
+          mountPath: /var/run/secrets/eks.amazonaws.com/serviceaccount
```

Comment:
the parameter name should be global.aws.irsa.enabled , based on ryan's comment..

Reply:
updated it!